.github/renovate.json

```diff
@@ -0,0 +1,10 @@
+{
+  "enabled": true,
+  "enabledManagers": ["docker-compose", "dockerfile", "github-actions", "pip_requirements", "pyenv"],
```

Comment:
Are we sure that default `dockerfile` manager is matching our dockerfiles? 
https://docs.renovatebot.com/modules/manager/dockerfile/#default-config

From that regexp I think it is not. Maybe we should add `fileMatch` for `dockerfile` as well?

Reply:
Yes, it is matching the dockerfiles : https://github.com/release-engineering/iib/pull/535